src/tile.cpp

```diff
@@ -396,94 +394,96 @@ void Tile::onUpdateTileItem(Item* oldItem, const ItemType& oldType, Item* newIte
 			}
 		}
 	} else if (oldItem->hasProperty(CONST_PROP_MOVEABLE) || oldItem->getContainer()) {
-		auto it = g_game.browseFields.find(this);
+		auto it = g_game.browseFields.find(getTile());
```

Comment:
Could this be `const auto`
```suggestion
		const auto it = g_game.browseFields.find(getTile());
```